lib/puma/events.rb

```diff
@@ -106,6 +109,7 @@ def parse_error(server, env, error)
     def ssl_error(server, peeraddr, peercert, error)
       subject = peercert ? peercert.subject : nil
       @stderr.puts "#{Time.now}: SSL error, peer: #{peeraddr}, peer cert: #{subject}, #{error.inspect}"
```

Comment:
Basically, I think _all_ error related output should go through this new interface. So no more `stderr.puts`, whatever we're logging today to stderr should be converted to this new format.

Reply:
Done